sdk/cosmos/tests.yml

```diff
@@ -226,6 +226,16 @@ jobs:
           ACCOUNT_CONSISTENCY: 'Session'
           DESIRED_CONSISTENCIES: '["Session"]'
           PREFERRED_LOCATIONS: null
+        # 15m 10s, expect passed
+        Session_Single_Master_Multi_Region_Tcp_MultiRegion:
+          ArmTemplateParameters: "@{ enableMultipleWriteLocations = $false; defaultConsistencyLevel = 'Session'; enableMultipleRegions = $true}"
+          DisplayName: Single Master Multi Region Multi-Region
+          OSVmImage: 'ubuntu-18.04'
```

Comment:
This needs to be updated to below, following @mitchdenny's changes to use our custom pools.

```
Pool: azsdk-pool-mms-ubuntu-1804-general
OSVmImage: MMSUbuntu18.04
```

Reply:
Thanks done